docs/bytes/Bytes.md

```diff
@@ -16,9 +16,9 @@
 
 The `Bytes` library has functions for working with bytes.
 
-#### Bytes, strings, numbers and endianness in Ethereum
+#### Bytes, strings, numbers and padding in Ethereum
 
-Ethereum uses the big endian format when working with strings/bytes, and little endian when working with other types (such as numbers and addresses). As an example, this is how we would store the string "abcd" in one full word (32 bytes):
```

Comment:
It still says "byte order" further down, otherwise it looks good, thanks!

Reply:
Replaced "byte order" with "padding rule"!